src/irc/core/modes.c

```diff
@@ -743,6 +743,7 @@ static char *get_nicks(IRC_SERVER_REC *server, WI_ITEM_REC *item,
 			    g_hash_table_lookup(optlist, "yes") == NULL) {
                                 /* too many matches */
 				g_string_free(str, TRUE);
+                                g_strfreev(matches);
```

Comment:
My fault, edited to tabs.  I usually make small commits on a project if I'm not sure of how they will be received.  You and @LemonBoy have been cool about this small off-by-1 change, thanks!  I have really enjoyed the project too :)